### PR TITLE
Some changes to app_localizations.dart

### PR DESCRIPTION
**Pull request by sandakersmann**
Some changes to app_localizations.dart:

- Add Norwegian Bokmål translation to app_localizations.dart
- Change Русский so it starts with capital letter
- Change Croatian to Хорватский in app_localizations.dart

### CODE REVIEW

File: lib/tools/app_localizations.dart [hunk 1]
```diff
@@ -20,16 +20,17 @@ class AppLocalizations {
 
   static const Map<String, String> availableLocales = {
     'en': 'English',
-    'hr': 'Croatian',
+    'hr': 'Хорватский',
```
Comment (bananenwilly) [inline lib/tools/app_localizations.dart:23]: @peerchemist

**Comment by bananenwilly** [comment]
Waiting for @peerchemist, I think this might be better pr'd against 0.8.2 branch, since the missing translation files would break main

**Comment by sandakersmann** [comment]
I don't know how to change branch other than making a new pull request. Can't you just put this on hold until the new version?

**Comment by bananenwilly** [comment]
I've consulted with @peerchemist  and decided to cherry pick your changes. 

https://github.com/peercoin/peercoin_flutter/commit/74164caaa641779a924da1ea0c709ccd46945c82
Thanks!

**Pull request closed without merging**